Dockerfile.full

```
@@ -47,37 +47,36 @@ COPY . .
 
 ENV VIRTUAL_ENV=/opt/venv
 RUN mkdir -p "$VIRTUAL_ENV" || true
-RUN chown spiderfoot:spiderfoot "$VIRTUAL_ENV"
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+RUN python -m venv "$VIRTUAL_ENV"
+
+ARG REQUIREMENTS=requirements.txt
+COPY "$REQUIREMENTS" requirements.txt
 
 RUN chown -R spiderfoot:spiderfoot /tools
-WORKDIR /tools
+RUN chown -R spiderfoot:spiderfoot "$VIRTUAL_ENV"
+RUN chown -R spiderfoot:spiderfoot "/home/spiderfoot"
 
 USER spiderfoot
 
-ARG REQUIREMENTS=requirements.txt
-COPY "$REQUIREMENTS" requirements.txt
-ENV PATH="$VIRTUAL_ENV/bin:$PATH"
-RUN python -m venv "$VIRTUAL_ENV"
 RUN pip install -U pip
 RUN pip install -r "$REQUIREMENTS"
 
 # Install Python tools
 RUN pip install dnstwist
 # CMSeeK
+WORKDIR /tools
 RUN git clone https://github.com/Tuhinshubhra/CMSeeK && cd CMSeeK \
-   && pip install -r requirements.txt && mkdir Results && cd /tools
+   && pip install -r requirements.txt && mkdir Results
 
 WORKDIR /home/spiderfoot
 
 EXPOSE 5001
 
-# Configure tool paths
-ENTRYPOINT python -c 'from spiderfoot import SpiderFootDb; \
+# Run the application
+CMD python -c 'from spiderfoot import SpiderFootDb; \
 db = SpiderFootDb({"__database": "spiderfoot.db"}, init=True); \
```

The `__database` key is supposed to represent the full file system path.

This should probably be:

```suggestion
db = SpiderFootDb({"__database": "/home/spiderfoot/.spiderfoot/spiderfoot.db"}, init=True); \
```